### PR TITLE
Update unison SHA1 checksum

### PR DESCRIPTION
**Pull request by matthewlmcclure**
Addresses a remaining issue relating to #35795.

### CODE REVIEW

**Comment by DomT4** [comment]
Can we bump this to SHA256 instead of SHA1? Thanks!

File: Library/Formula/unison.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Unison < Formula
   desc "Unison file synchronizer"
   homepage 'http://www.cis.upenn.edu/~bcpierce/unison/'
   url 'http://www.seas.upenn.edu/~bcpierce/unison//download/releases/stable/unison-2.48.3.tar.gz'
-  sha1 '74f1c087ee49dc1db4680ad779280f7333d5c968'
```
Comment (vszakats) [inline Library/Formula/unison.rb]: Would you update both `url` and `homepage` to https? Both addresses support it.

File: Library/Formula/unison.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@ class Unison < Formula
...
+  sha1 '2a0cfc95e95b9e9457c39faa4fe5b8184023cd0c'
```
Comment (mikemcquaid) [inline Library/Formula/unison.rb]: Update this to `sha256`, thanks.

File: Library/Formula/unison.rb (hunk 1 above)
Comment (matthewlmcclure) [inline Library/Formula/unison.rb]: Done.

File: Library/Formula/unison.rb (hunk 2 above)
Comment (matthewlmcclure) [inline Library/Formula/unison.rb]: Done.

**Comment by matthewlmcclure** [comment]
@DomT4,

> Can we bump this to SHA256 instead of SHA1? Thanks!

Done.

File: Library/Formula/unison.rb (hunk 1 above)
Comment (vszakats) [inline Library/Formula/unison.rb]: Thanks!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/fccf6477f0e521276a7b2a85855f651046751aa3. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Just for future reference the preferred commit message for non-version bumps is styled `unison: update checksum, use https` or similar, Thanks!

**Pull request closed without merging**